task06/routes/product.route.ts

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+import { Request, Response } from 'express';
+import { getProducts, getProduct } from '../controller/product.controller';
+import { authenticateUser } from '../controller/user.controller';
+
+const router = express.Router();
+router.use(authenticateUser);
+router.get('/', (req: Request, res: Response) => {
+  getProducts(req, res);
+});
```

Comment:
The same regarding code simplifications.

Reply:
Fixed